f5_openstack_agent/lbaasv2/drivers/bigip/system_helper.py

```diff
@@ -69,138 +67,37 @@ def get_minor_version(self, bigip):
         return version
 
     def get_version(self, bigip):
-        devices = bigip.tm.cm.devices.get_collection()
+        d = Device()
+        devices = d.get_collection(bigip)
```

Comment:
seems similar here and other places as well . pls check .

Reply:
Done.